content-src/components/PocketLoggedInCta/_PocketLoggedInCta.scss

```diff
@@ -27,6 +27,7 @@
   .cta-text {
     font-weight: normal;
     font-size: 13px;
+    line-height: 1.230769231; // (16 / 13) – 16px computed
```

Comment:
"more recommendations" is no longer aligned: 
![align](https://user-images.githubusercontent.com/197334/47858494-9485f100-ddc2-11e8-8d25-166fd68d0a1c.png)
I think it just needs a matching line-height at the least.